src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -96,6 +96,13 @@ public Conversation getConversationWithTitle(String title) {
     return null;
   }
 
+  /** Accesses the conversations stored
+   *  @return the size of conversations list
+   */
+  public Integer countTotalConversations() {
```

Comment:
Just out of curiosity I was wondering why this returns an "Integer" instead of an int, don't think i've ever really seen it done like that.

Reply:
That's actually a good question, since I started using Java I've always used Integer instead of int, String instead of string, etc. After doing some research I found this, the difference is that Integer is a class and acts as a wrapper for an int, an Integer can have a null value whereas an int has 0 instead of null.

https://www.quora.com/What-is-the-difference-between-an-integer-and-int-in-Java
https://stackoverflow.com/questions/423704/which-one-to-use-int-or-integer